contracts/test/Tracker.t.sol

```diff
@@ -58,6 +60,44 @@ contract TrackerTest is Test {
         tracker.settle(0);
         (,,,,,,, bool settled) = tracker.challenges(0);
         require(settled, "Settle failed");
-        require(tracker.balances(address(this)) == 0.0001 ether, "Settle failed");
+        require(tracker.getClaimableAmount(0, address(this)) == 0.0001 ether, "Settle failed");
+    }
+
+    function test_withdraw(uint256 timestamp) public {
+        vm.assume(timestamp > 0 && timestamp < 1e20);
+        commonJoinAndCheckIn(timestamp);
+
+        vm.warp(block.timestamp + 1001);
+        tracker.settle(0);
+        uint256 beforeBalance = underlying.balanceOf(address(this));
+        tracker.withdraw(0);
+        uint256 afterBalance = underlying.balanceOf(address(this));
+        require(afterBalance - beforeBalance == PER_USER_STAKE, "withdraw failed");
+    }
+
+    function test_withdraw_failed_user(address failedUser, uint256 timestamp) public {
+        vm.assume(timestamp > 0 && timestamp < 1e20);
+        commonJoinAndCheckIn(timestamp);
+
+        vm.startPrank(failedUser);
+        underlying.mint(failedUser, PER_USER_STAKE);
+        uint256 failedUserBalance = underlying.balanceOf(failedUser);
+        underlying.approve(address(tracker), PER_USER_STAKE);
+        tracker.join(0);
```

Comment:
we should update the contract so that the ID doesn't start with 0, looks very weird.